src/management-system-v2/app/(dashboard)/[environmentId]/executions/[processId]/element-status.tsx

```diff
@@ -0,0 +1,282 @@
+import { ReactNode } from 'react';
+import { Alert, Checkbox, Image, Progress, ProgressProps, Space, Typography } from 'antd';
+import { ClockCircleFilled } from '@ant-design/icons';
+import React from 'react';
+import { statusToType } from './instance-helpers';
+import { convertISODurationToMiliseconds, getMetaDataFromElement } from '@proceed/bpmn-helper';
+import { generateRequestUrl } from '@/lib/engines/endpoints';
+import { DisplayTable, RelevantInstanceInfo } from './instance-info-panel';
+
+function transformMilisecondsToTimeFormat(milliseconds: number | undefined) {
+  if (!milliseconds || milliseconds < 0 || milliseconds < 1000) return;
+
+  const days = Math.floor(milliseconds / (3600000 * 24));
+  milliseconds -= days * (3600000 * 24);
+  const hours = Math.floor(milliseconds / 3600000);
+  milliseconds -= hours * 3600000;
+  // Minutes part from the difference
+  const minutes = Math.floor(milliseconds / 60000);
+  milliseconds -= minutes * 60000;
+  //Seconds part from the difference
+  const seconds = Math.floor(milliseconds / 1000);
+  milliseconds -= seconds * 1000;
+
+  // Will display time in 10:30:23 format
+  return `${days} Days, ${hours}h, ${minutes}min, ${seconds}s`;
+}
+
+export function ElementStatus({ info }: { info: RelevantInstanceInfo }) {
+  const statusEntries: ReactNode[][] = [];
+
+  const isRootElement = info.element && info.element.type === 'bpmn:Process';
+  const metaData = getMetaDataFromElement(info.element.businessObject);
+  const token = info.instance?.tokens.find((l) => l.currentFlowElementId == info.element.id);
+  const logInfo = info.instance?.log.find((logEntry) => logEntry.flowElementId === info.element.id);
+
+  // Element image
+  if (metaData.overviewImage)
+    statusEntries.push([
+      'Image',
+      <div
+        style={{
+          width: '75%',
+          display: 'flex',
+          justifyContent: 'center',
+          margin: 'auto',
+          marginTop: '1rem',
+        }}
+      >
+        <Image
+          src={generateRequestUrl(
```

Comment:
Should this work when watching an instance of a process that is not stored in the MS?